R/init.R

```diff
@@ -46,6 +46,9 @@
 #'   See the vignette for an example. However, [ui_teal_with_splash()]
 #'   is then preferred to this function.
 #'
+#' @note If you pass a module of class `"teal_module_landing"` in `modules` parameter, `teal` will not create a tab for
+#' this module.
+#'
```

Comment:
Create a section called "Module classes" and describe it there. It may grow in the future.

Reply:
I actually will remove this note at all. This is documented in `tm_landing_popup` manual

Reply:
3da289f71c810c7bc83d25de310fb66451862c91